public/member_ops.js

```diff
@@ -180,9 +209,77 @@ function handle_members(event) {
         }
       });
     } else if (event.target === createFact) {
-      // Your code goes here!
+      var dataObject = {
+        text: factText.value,
+        likes: likes.value,
+      };
+      let headers = { "Content-Type": "application/json" };
+      let csrf_cookie = getCookie("CSRF-TOKEN");
+      if (csrf_cookie) {
+        headers["X-CSRF-Token"] = csrf_cookie;
+      }
+      fetch(`${members_path}/${memberId4.value}/facts`, {
+        method: "POST",
+        headers: headers,
+        body: JSON.stringify(dataObject),
+      }).then((response) => {
+        if (response.status === 201) {
+          response.json().then((data) => {
+            resultsDiv.innerHTML = "";
+            let parag = document.createElement("P");
+            parag.textContent = `Fact created successfully: ${JSON.stringify(data)}`;
+            resultsDiv.appendChild(parag);
+          });
+        } else {
+          response
+            .json()
+            .then((data) => {
+              alert(
+                `Return code ${response.status} ${response.statusText} ${JSON.stringify(data)}`,
+              );
+            })
+            .catch((error) => {
+              console.log(error);
+              alert(error);
+            });
+        }
+      });
     } else if (event.target === updateFact) {
-      // Your code goes here!
+      var dataObject = {
+        text: factText2.value,
+        likes: likes2.value,
+      };
```

Comment:
Same issue here as above -- you just need to make a small change to the shape of the data here. Ping me in slack if you want to discuss, or we can review it in our 1:1 Monday.